contracts/wrapper/NameWrapper.sol

```diff
@@ -517,6 +535,11 @@ contract NameWrapper is
         ) {
             revert OperationProhibited(node);
         }
+
+        if (expiry < block.timestamp) {
```

Comment:
I don't think this is necessary either, because anything reading the fuses will treat them as 0 if the name is expired.